resources/schema/spidacalc/client/wire_height_calculator.schema

```diff
@@ -0,0 +1,17 @@
+{
+    "type": "object",
+    "id": "#/spidacalc/client/wire_height_calculator.schema",
+    "description": "Contains information on how to query a distance above ground for a component",
+    "properties": {
+      "name": {
```

Comment:
name and type are required; there are three valid values for type; wireState is required if it is a WireStateHeightCalculator and not allowed for other types